src/main/webapp/lecture-view.html

```diff
@@ -3,5 +3,5 @@
 
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
-<script src="view.js" defer></script>
-<p>View Lecture Page</p>
+<script src="video.js" defer></script>
```

Comment:
We should load both the view and video scripts.  View is the one that parses the lecture id and initializes the other sections
```suggestion
<script src="view.js" defer></script>
<script src="video.js" defer></script>
```

Reply:
fixed